scripts/base/frameworks/telemetry/main.zeek

```diff
@@ -0,0 +1,498 @@
+##! Module for recording and querying metrics. This modules wraps
+##! the lower-level telemetry.bif functions.
+
+module Telemetry;
+
+export {
+	type labels_table: table[string] of string;
+	type labels_vector: vector of string;
```

Comment:
Add short comments here as well, rule of thumb is that everything in the export section should be documented.